programs/drift/src/controller/orders.rs

```diff
@@ -1257,6 +1258,7 @@ pub fn fill_perp_order(
         now,
         slot,
         fill_mode,
+        user_can_skip_duration,
```

Comment:
right now this is always false is `amm_can_skip_duration` is false, idt that makes sense?